src/rez/config.py

```diff
@@ -583,7 +583,7 @@ def _get_plugin_completions(prefix_):
                 keys += _get_plugin_completions('')
             return keys
 
-    def _uncache(self, key=None):
+    def _uncache(self, key=None, keep_plugins=False):
```

Comment:
What is the relevance of `keep_plugins`? This seems like a change that could have unforeseen consequences, and I can't figure out how it's related.

Reply:
This is the fix for a bug that was not triggered because of the layout of the tests. Compare:
https://github.com/nerdvegas/rez/pull/739/commits/29bc81af6d7aa7ee0de2c4eeb668436088602813#r331578108